tests/unit_tests/models/qwen/test_qwen3_bridge.py

```
@@ -505,3 +505,50 @@ def test_qwen3_no_qkv_bias_mapping(self):
+        "mtp.layers.0.enorm.weight",
+        "mtp.layers.0.hnorm.weight",
+        "mtp.layers.0.final_layernorm.weight",
+        "mtp.layers.0.transformer_layer.self_attention.linear_qkv.layer_norm_weight",
+        "mtp.layers.0.transformer_layer.self_attention.q_layernorm.weight",
+        "mtp.layers.0.transformer_layer.self_attention.k_layernorm.weight",
+        "mtp.layers.0.transformer_layer.self_attention.linear_proj.weight",
+        "mtp.layers.0.transformer_layer.self_attention.linear_qkv.weight",
+        "mtp.layers.0.transformer_layer.mlp.linear_fc1.layer_norm_weight",
+        "mtp.layers.0.transformer_layer.mlp.linear_fc1.weight",
+        "mtp.layers.0.transformer_layer.mlp.linear_fc2.weight",
+    ]
```

Ruff (0.15.10)

[warning] Mutable default value for class attribute

(RUF012)